portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function showInfo(number) {
+  var elementId = "info-" + number;
+  console.log(elementId);
+  var infoDiv = document.getElementById(elementId);
+  var buttonDiv = document.getElementById("div-" + number);
+
+  console.log("Button has been pressed. Display of element is " + infoDiv.style.display);
+  
+  if (infoDiv.style.display === "none") {
+    infoDiv.style.display = "block";
+    buttonDiv.style.display = "none";
+  }
 }
+
+function showGreeting() {
+  var greetings = ["Hello!", "Buna ziua!", "Buenos Dias!", "Buongiorno!", "Bonjour!", "こんにちは!", "Χαίρετε!", "Dzień dobry!"];
+  var greetElement = document.getElementById("greeting");
+
+  var x = Math.floor(Math.random() * 10) % 8;
+  greetElement.innerText = greetings[x];
+}
+
+window.onload = function() {
+  
+  /*
+  function showFavourites(elementId) {
+    console.log("Show element with ID " + elementId);
+    var infoDiv = document.getElementById(elementId);
+    infoDiv.style.display = "block";
+  }
+
+  function hideFavourites(elementId) {
+    console.log("Hide element with ID " + elementId);
+    var infoDiv = document.getElementById(elementId);
+    infoDiv.style.display = "none";
+  }*/
+
+  for (var i = 1; i <= 3; i++) {
```

Comment:
Can you make this code more universal - i.e. to work on all scroll- ids regardless of how many of them there are?